.github/workflows/build-macos.yml

```diff
@@ -100,3 +100,28 @@ jobs:
       - name: Smoke tests
         shell: bash
         run: make smoketest TINYGO=$(PWD)/build/tinygo AVR=0
+  test-macos-homebrew:
+    name: Test on MacOS with Homebrew
```

Comment:
Perhaps change this to `homebrew-install` to match the format for the names of all the other workflows?